packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/test/helpers/goldens_file_comparator.dart

```diff
@@ -0,0 +1,48 @@
+import 'dart:developer';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+class GoldensFileComparator extends LocalFileComparator {
+  static const double _kGoldenDiffTolerance = 0.0041;
```

Comment:
Why we need `0.0041` tolerance ?

Reply:
@StanevPrime  Copied from the booking app. Any suggestions for the minimal tolerance?

Reply:
0, unless is needed more

Reply:
While having some small margin for error is always a good thing to have, we can go with `0`. Although judging from previous experience, this has always been a pain point in the CI.